data_steward/tools/consolidated_achilles_report/query_reports.py

```diff
@@ -11,35 +11,61 @@
   MAX(timestamp) AS upload_timestamp,
   CONCAT('gs://{drc_bucket}', MAX(SUBSTR(protopayload_auditlog.resourceName, 49))) as file_path
 FROM
-  `{app_id}.GcsBucketLogging.cloudaudit_googleapis_com_data_access_2018*` l
+  `{app_id}.GcsBucketLogging.cloudaudit_googleapis_com_data_access_2019*` l
```

Comment:
Maybe we can change the date portion to be a variable.  Then this can be more easily updated in the future.

```
`{app_id}.GcsBucketLogging.cloudaudit_googleapis_com_data_access_{year}*` l
```